autoload/peekaboo.vim

```diff
@@ -232,7 +234,15 @@ function! peekaboo#aboo()
     if visualmode
       normal! gv
     endif
-    call s:feed(cnt, mode, reg, rest)
+    if a:insert_mode
+      augroup peekaboo_hide
+        au!
+        au TextChangedI * let [&showtabline, &laststatus] = [stl, lst] | call s:close() | redraw
```

Comment:
I'm not so sure about this (resetting `&showtabline` and `&laststatus` to these variables inside the autocommand), but I see no errors. Maybe they should be made script variables?